src/notebooks/finance/interventions/pleiss.ipynb

```diff
@@ -189,22 +248,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Learn parameters to equalize odds and apply to create a new dataset\n",
+    "# Learn parameters to equal opportunity and apply to create a new dataset\n",
     "cpp = CalibratedEqOddsPostprocessing(\n",
     "    privileged_groups=privileged_groups,\n",
     "    unprivileged_groups=unprivileged_groups,\n",
     "    cost_constraint=cost_constraint,\n",
-    "    seed=42,\n",
+    "    seed=np.random.seed(),\n",
```

Comment:
I think fixing the seed to make sure results are reproducible is probably favourable here?